ouroboros-consensus-cardano/src/shelley/Ouroboros/Consensus/Shelley/Ledger/Ledger.hs

```diff
@@ -320,6 +321,32 @@ instance ShelleyCompatible proto era => UpdateLedger (ShelleyBlock proto era)
 type instance TxIn (LedgerState (ShelleyBlock proto era)) = SL.TxIn
 type instance TxOut (LedgerState (ShelleyBlock proto era)) = Core.TxOut era
 
+-- | We use this newtype only to alter MemPack serialization. LSM trees use an
+-- index that looks at the first 8 bytes so it is important to put the index
+-- first such that we avoid all TxIns from the same tx to imply a collision in
+-- the LSM index.
```

Comment:
```suggestion
-- | We use this newtype only to alter MemPack serialization. LSM trees use an
-- index that looks at the first 8 bytes, so it is important to put the index
-- first to avoid all TxIns from the same tx to cause a collision in
-- the LSM index.
```